ouroboros-consensus/src/Ouroboros/Consensus/Mempool/API.hs

```diff
@@ -191,19 +191,19 @@ data Mempool m blk idx = Mempool {
 -- even for tx sequence numbers returned in previous snapshots. This happens
 -- when the transaction has been removed from the mempool between snapshots.
 --
-data MempoolSnapshot txid tx idx = MempoolSnapshot {
+data MempoolSnapshot blk idx = MempoolSnapshot {
     -- | Get all transactions in the mempool snapshot along with their
     -- associated ticket numbers (oldest to newest).
-    getTxs :: [(txid, tx, idx)]
+    getTxs      :: [(GenTxId blk, GenTx blk, idx)]
```

Comment:
Not for this PR, but now that this is a snapshot. I'd rename these things to avoid the "get" prefix. They're not action in STM. e.g. `mempoolSnapshotTxs`

Reply:
Ok, I might do that as part of #739.